doc/deps.md

```diff
@@ -47,23 +49,43 @@ The standard Ubuntu Xenial package repository contains only LLCM/Clang version
 6, which is not sufficient for CodeCompass, as at least version 7.0 is
 required.  Therefore LLVM and Clang should be installed from the official LLVM
 repositories:
-
 ```bash
 sudo deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main
 sudo deb-src http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main
-
+```
+The default-jdk repository ponts to teh openjdk-11 version of java, whih
+is not sufficent for Thrift, therefore its uninstall is required:
+```bash
+sudo apt remove default-jdk default-jdk-headless default-jre \
+  default-jre-headless openjdk-11-jdk  openjdk-11-jdk-headless \
+  openjdk-11-jre openjdk-11-jre-headless
+```
+Installation command for all the required packages of CodeCompass:
+```bash
 sudo apt-get install git cmake make g++ libboost-all-dev \
   llvm-7-dev libclang-7-dev odb libodb-dev \
-  default-jdk libssl-dev libgraphviz-dev libmagic-dev libgit2-dev ctags \
+  openjdk-8-jdk openjdk-8-dbg openjdk-8-jre \
+  openjdk-8-jdk-headless openjdk-8-dbg libssl-dev \
+  libgraphviz-dev libmagic-dev libgit2-dev ctags \
   libgtest-dev npm nodejs-legacy
 ```
 
 #### Ubuntu 18.04 LTS
 
+The default-jdk repository ponts to teh openjdk-11 version of java, whih
+is not sufficent for Thrift, therefore its uninstall is required:
```

Comment:
This part of the PR should be removed completely for multiple reasons:
- Downgrading to JDK8 and requiring to uninstall JDK11 on a modern OS like Ubuntu 18.08 is a bad idea, which would make using CodeCompass in a production environment difficult.
- Despite the configuration error message `Could not determine java version from '11.0.4'.`, Thrift compiles and works properly, as it was pointed out by @bruntib in #349. It was also noted that maybe @zomen2 has a proper solution for it?